### ISSUE TITLE
[a11y]: Snackbar should not disappear until the user takes an action when accessibility controls are turned on

### ISSUE THREAD

**Issue by DaveShuckerow** [flutter/flutter#16243]
In the Bottom App Bar demo, pressing the Floating Action Button shows a snackbar.  I noticed this while writing #16238

For this to be accessible to users with a screen reader or other accessibility feature turned on, the snackbar should remain on the screen until the user takes another action.

This should be a change we make at the Snackbar level.

@goderbauer

**Comment by goderbauer** [comment]
Internal discussion of this: yaqs/5408254254645248
